### PR TITLE
Changes to cluster config editor view. Added dropdown lists

### PR DESCRIPTION
**Pull request by KorolS**
#48 
Some improvement to cluster config editor

### CODE REVIEW

File: source/DG.HostApp/Components/ClusterConfigEditor/ApplicationTypeSourcesCard.razor [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            <th> </th>
+            <th> </th>
+        </tr>
+        @foreach (var applicationTypeSource in ClusterConfig.ClusterDefinition.ApplicationTypesSources)
+
+        {
+    <tr>
+        <td>@applicationTypeSource.Name</td>
+        <td>@applicationTypeSource.Path </td>
+        <td>@applicationTypeSource.PathType"</td>
```
Comment (motpfofs) [inline source/DG.HostApp/Components/ClusterConfigEditor/ApplicationTypeSourcesCard.razor:19]: Should we add or delete one of "?

File: source/DG.HostApp/Components/ClusterConfigEditor/ApplicationTypeSourcesCard.razor [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <td> <button class=" btn btn-outline-primary" @onclick="() => SetTypeSource(applicationTypeSource) ">Edit</button></td>
```
Comment (motpfofs) [inline source/DG.HostApp/Components/ClusterConfigEditor/ApplicationTypeSourcesCard.razor:20]: Could we make Edit button to toggle edit menu?

**Review by motpfofs — APPROVED** [review]
Maybe we should consider renaming buttons in the edit form to "OK", "Cancel"? (Like "Source B" edit form in attachment).

image

File: source/DG.HostApp/Components/ClusterConfigEditor/ApplicationTypeSourcesCard.razor (hunk 1 above)
Comment (KorolS) [reply source/DG.HostApp/Components/ClusterConfigEditor/ApplicationTypeSourcesCard.razor:19]: Yes, we should delete it

File: source/DG.HostApp/Components/ClusterConfigEditor/ApplicationTypeSourcesCard.razor (hunk 2 above)
Comment (KorolS) [reply source/DG.HostApp/Components/ClusterConfigEditor/ApplicationTypeSourcesCard.razor:20]: Edit button expands edit menu, when we click on it, and collapses it when we click submit button (we can rename it to "Ok" as you suggested).

**Pull request merged**